neonvm-kernel/README.md

```diff
@@ -26,9 +26,10 @@ with the following sequence of actions:
    ```
 2. Then, inside the container, run:
    ```sh
+   export KERNEL_ARCH=arm # x86_64 or arm
    cd linux-$NEW_VERSION
    cp /host/linux-config-6.1.92 .config # Copy current config in
```

Comment:
only thing I'd note is that this `linux-config-6.1.92` path is outdated as well. Up to you how/if you want to fix that.